docs/modules/ROOT/pages/advanced-scripting.adoc

```diff
@@ -537,73 +537,6 @@ Fun 1
 
 == Output
 
-=== Type of CLC Output
-
-CLC supports two kinds of output: structured and "unnecessary".
-Let's talk about them over an example.
-
-`clc demo generate-data` command creates data in a Map or to the terminal.
-Let's run it and analyze its output:
-
-[source,bash]
-----
-clc demo generate-data wikipedia-event-stream --preview --yes --max-values 1
-----
-
-Running that command generates output similar to the following:
-----
-Following mapping will be created when run without preview:
```

Comment:
is this part of the output?

When run without the `--preview` option, the following mapping is created: